From abd4bd1128b72eff96c89d79727a6c8f8d5a3e98 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Wed, 20 Apr 2016 11:22:41 -0700
Subject: [PATCH 23/28] Do not use gold to link.

If acl/attr are linked without gold, then this won't compile.
---
 configure.ac | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 0b79888..220c9a8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -239,8 +239,7 @@ CC_CHECK_FLAGS_APPEND([with_ldflags], [LDFLAGS], [\
         -Wl,--as-needed \
         -Wl,--no-undefined \
         -Wl,-z,relro \
-        -Wl,-z,now \
-        -Wl,-fuse-ld=gold])
+        -Wl,-z,now])
 AC_SUBST([OUR_LDFLAGS], "$with_ldflags $sanitizer_ldflags")
 
 AC_CHECK_SIZEOF(pid_t)
-- 
2.9.0

